.github/workflows/build.yml

```diff
@@ -0,0 +1,85 @@
+name: Build and test
+on:
+  push:
+    branches:
+      - main
+    paths-ignore:
+      - '**/*.md'
+  pull_request:
+    branches:
+      - main
+    paths-ignore:
+      - '**/*.md'
+  release:
```

Comment:
This will only trigger whenever we cut the release from their UI. How about if someone pushes the tag directly.